tests/functional/BaseCase.php

```diff
@@ -240,4 +244,37 @@ static function (array $agg, StorageObject $file) {
 
         return $result;
     }
+
+    protected function getWorkspaceId(): string
+    {
+        return substr(md5($this->getName()), 0, 9) . '_test';
```

Comment:
ked to bola cela md5 tak sa to na niektorych miestach nevoslo je tam limit na 30 znakov